### PR TITLE
Explicitly set static crt

### PR DESCRIPTION
**Pull request by liranringel**
Will help to resolve https://github.com/rust-lang/rust/issues/40246

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -232,13 +242,15 @@ impl Config {
                                          .debug(false)
                                          .target(&target)
                                          .host(&host)
+                                         .static_crt(self.static_crt.unwrap_or(false))
```
Comment (alexcrichton) [inline src/lib.rs]: For maximal safety to trigger the `None` fallback logic in gcc-rs, could this only be configured if `static_crt` is explicitly set in cmake?

Comment (liranringel) [inline src/lib.rs]: Oh right!

File: src/lib.rs [hunk 2]
```diff
@@ -227,19 +237,26 @@ impl Config {
...
+            .opt_level(0)
+            .debug(false)
+            .target(&target)
+            .host(&host);
+        if let Some(static_crt) = self.static_crt {
+            c_cfg.static_crt(static_crt);
+            cxx_cfg.static_crt(static_crt);
+        }
+
+        let c_compiler = gcc::Config::new().get_compiler();
```
Comment (alexcrichton) [inline src/lib.rs]: Shouldn't this be using `c_cfg` and `cxx_cfg` from above?

Comment (liranringel) [inline src/lib.rs]: Oops, fixed that

**Comment by alexcrichton** [comment]
Excellent! I'll make a release now

**Pull request merged**